text/0756-helper-default-manager.md

```diff
@@ -108,6 +108,26 @@ class A {
 }
 ```
 
+#### Example with consuming tracked data defined outside of the helper
+
+This works with the `helper` and `Helper` from `@ember/component/helper`, as well as plain functions.
+
+```hbs
+<output>{{this.multiply 4}}</output>
+
+<button {{on 'click' this.increment}}>Increment</button>
```

Comment:
This isn't really technically a helper per-se, the first example is probably good enough